OTAnalytics/application/datastore.py

```diff
@@ -12,14 +14,56 @@ def parse(self, file: Path) -> list[Track]:
         pass
 
 
-@dataclass(frozen=True)
-class Video:
-    path: Path
+class VideoReader(ABC):
```

Comment:
Not sure if this should stay here 🤔

Reply:
I will check this with the code I wrote on the other PR.